src/analyzer.rs

```diff
@@ -172,6 +174,7 @@ pub struct DynamicDependencyDescriptor {
   pub argument: DynamicArgument,
   /// The range of the argument.
   pub argument_range: PositionRange,
+  pub range: PositionRange,
```

Comment:
I just realized this breaks the deserializtion of information we store on JSR. We'll need to think of a solution for this.